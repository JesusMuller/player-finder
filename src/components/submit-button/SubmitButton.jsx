import React from 'react';
import { string } from 'prop-types';
import { submitButton } from './SubmitButton.module.scss';

const SubmitButton = ({ buttonText }) => (
    <input className={submitButton} type="submit" value={buttonText} />
);

SubmitButton.propTypes = {
    buttonText: string
};

export default SubmitButton;
