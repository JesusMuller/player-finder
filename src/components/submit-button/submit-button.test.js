import React from 'react';
import { shallow } from 'enzyme';
import SubmitButton from './submit-button';

describe('<SubmitButton />', () => {
    it('Should match snapshot', () => {
        const wrapper = shallow(<SubmitButton buttonText="Search" />);
        expect(wrapper).toMatchSnapshot();
    });
});
