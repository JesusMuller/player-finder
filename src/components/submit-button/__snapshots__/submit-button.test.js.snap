// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubmitButton /> Should match snapshot 1`] = `
<input
  className="submitButton"
  type="submit"
  value="Search"
/>
`;
